client/components/feedback/style.scss

```diff
@@ -4,6 +4,11 @@
 	display: flex;
 	position: fixed;
 	z-index: 100;
+
+	> div:last-child:not(.crowdsignal-forms-feedback__trigger) {
```

Comment:
What is this and why can't we just give it a proper class name?

Reply:
We need the rounded corners at the bottom of the modal, and the new notice that goes there is just appended to the popover. So, whatever div is last, excluding the `trigger`, should have this roundness.

The notice does have a class `.crowdsignal-forms-feedback__closed-notice`, but if we just put the rounded corners on both it doesn't look good, like a double smile on a face.